test/runtime/gbt/comm/ugni/SIGBUS-array-alloc.good

```diff
@@ -0,0 +1 @@
+SIGBUS-array-alloc.chpl:15: error: Out of memory allocating "array elements"
```

Comment:
Do you want the message to call attention to it being a hugepage allocation failure specifically?  (If so, is the particular hugepage size of the region available in the table the signal handler is consulting?)

Reply:
No, it should look just like any other allocation failure.  Whether or not a particular allocation was satisfied out of hugepage or non-hugepage memory is not something we want Chapel programmers or programs to care about.